api/src/test/java/org/pmiops/workbench/api/ConceptSetsControllerTest.java

```diff
@@ -174,8 +174,10 @@ public class ConceptSetsControllerTest {
 
   private static final String USER_EMAIL = "bob@gmail.com";
   private static final String WORKSPACE_NAMESPACE = "ns";
-  private static final String WORKSPACE_NAME = "name";
```

Comment:
Sometimes "name" was used as the display name, and sometimes as the terra name, so parts of this test may not have testing the right things.  Now they are clearly distinct